internal/server/config.go

```diff
@@ -621,6 +624,15 @@ func (s Server) loadConfig(config Config) error {
 			Resource: "infra",
 		})
 
+		org, err := data.GetOrganization(tx, data.ByName(config.OrganizationName))
+		if errors.Is(err, internal.ErrNotFound) {
+			org = &models.Organization{Name: config.OrganizationName, Domain: config.OrganizationDomain}
```

Comment:
We should fail if the organization domain has a port (to match the middleware which strips it)

Reply:
hmm! interesting question. what if you intend to test it with different ports. maybe I need to only strip it if there are no matches.